errors/mappers/pqerrors/pqerrors.go

```diff
@@ -0,0 +1,117 @@
+package pqerrors
+
+import (
+	"context"
+
+	"github.com/lib/pq"
+	"google.golang.org/grpc/codes"
+
+	"github.com/infobloxopen/atlas-app-toolkit/errors"
+)
+
+const (
+	msgForeignKeyViolation = "Cannot insert The '%s' object as it does not refer to a valid '%s' object."
+	msgRestrictViolation   = "Cannot update or delete an '%s' object as it is referenced by a '%s' object."
```

Comment:
"Cannot update or delete an object '%s' ... ?

Reply:
here '%s' stands for type of object. so Cannot update or delete an 'Network' object.